Bottom: 2b3c50d0ed0a3cca4a4165e861e8e537a208ae12
Top:    32f045464ccbc95c66aa8185ff9a46e6829ec2ee
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-22 20:31:13 -0400

Refresh of dspeak-audio

---

diff --git a/sound/soc/fsl/Makefile b/sound/soc/fsl/Makefile
index 159718b..7695a0c 100644
--- a/sound/soc/fsl/Makefile
+++ b/sound/soc/fsl/Makefile
@@ -18,5 +18,5 @@ obj-$(CONFIG_SND_SOC_MPC5200_AC97) += mpc5200_psc_ac97.o
 # MPC5200 Machine Support
 obj-$(CONFIG_SND_MPC52xx_SOC_EFIKA) += efika-audio-fabric.o
 obj-$(CONFIG_SND_MPC52xx_SOC_PCM030) += pcm030-audio-fabric.o
-obj-$(CONFIG_SND_MPC52xx_SOC_DSPEAK01) += dspeak01_fabric.o
+obj-$(CONFIG_SND_MPC52xx_SOC_DSPEAK01) += dspeak01-audio-fabric.o
 
diff --git a/sound/soc/fsl/dspeak01_fabric.c b/sound/soc/fsl/dspeak01-audio-fabric.c
similarity index 100%
rename from sound/soc/fsl/dspeak01_fabric.c
rename to sound/soc/fsl/dspeak01-audio-fabric.c
